### ISSUE TITLE
`aria-setsize` is set incorrectly for virtual items.

### ISSUE THREAD

**Issue by mannycarrera4** [Workday/canvas-kit#3529]
## 🐛 Bug Report

Given this storybook: https://stackblitz.com/edit/ef8p2dea?file=src%2FDemo.tsx

`aria-setsize` is set to the **total pixel value** which is incorrect and it makes it seem like the virtual list is much larger than what it should be.
